node/src/effect.rs

```diff
@@ -132,7 +132,7 @@ use requests::{
 use self::announcements::BlocklistAnnouncement;
 
 /// A resource that will never be available, thus trying to acquire it will wait forever.
-static UNOBTAINIUM: Lazy<Semaphore> = Lazy::new(|| Semaphore::new(0));
+static UNOBTAINABLE: Lazy<Semaphore> = Lazy::new(|| Semaphore::new(0));
```

Comment:
That one wasn't by mistake, I think. :smile:

Reply:
Yes, I just thought it might antagonise folks and seem unprofessional if the related `panic!` ever did end up being hit.